config/samples/controlplane-konnect-extension.yaml

```
@@ -0,0 +1,77 @@
+# Ensure that you create a secret containing your cluster certificate before applying this
+# kubectl create secret tls konnect-client-tls -n kong --cert=./tls.crt --key=./tls.key
+# ---
+# apiVersion: v1
+# kind: Secret
+# metadata:
+#   name: konnect-client-tls
+# type: kubernetes.io/tls
+# stringData:
+#   tls.crt: |
+#     -----BEGIN CERTIFICATE-----
+#     MIIDhDCCAm6gAwIBAgIBATALBgkqhkiG9w0BAQ0wLDEqMAkGA1UEBhMCVVMwHQYD...
+#     zy5lW2IG5AjNDV8VBCthVj5j1UENTVi4rLhu8j/kfb9gNhvqaN8UcA==
+#     -----END CERTIFICATE-----
+#   tls.key: |
+#     -----BEGIN PRIVATE KEY-----
+#     MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQChCERwsegdWSnS...
+#     WSK9kndNKpFI4vPvuw6j2JJl
+#     -----END PRIVATE KEY-----
+# ---
+kind: KonnectExtension
+apiVersion: konnect.konghq.com/v1alpha1
+metadata:
+  name: my-konnect-config
+  namespace: default
+spec:
+  konnectControlPlane:
+    controlPlaneRef:
+      type: konnectID
+      konnectID: a6554c4c-79a6-4db7-b7a4-201c0cf746ba 
+  dataPlaneClientAuth:
+    certificateSecret:
+      provisioning: Manual
+      secretRef:
+        name: konnect-client-tls
+  konnect:
+    authRef:
+      name: konnect-api-auth-dev-1
+---
+kind: KonnectAPIAuthConfiguration
+apiVersion: konnect.konghq.com/v1alpha1
+metadata:
+  name: konnect-api-auth-dev-1
+  namespace: default
+spec:
+  type: token
+  token: kpat_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  serverURL: us.api.konghq.com
```

APIAuth is required by KE so let's switch the order here

```suggestion
kind: KonnectAPIAuthConfiguration
apiVersion: konnect.konghq.com/v1alpha1
metadata:
  name: konnect-api-auth-dev-1
  namespace: default
spec:
  type: token
  token: kpat_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
  serverURL: us.api.konghq.com
---
kind: KonnectExtension
apiVersion: konnect.konghq.com/v1alpha1
metadata:
  name: my-konnect-config
  namespace: default
spec:
  konnectControlPlane:
    controlPlaneRef:
      type: konnectID
      konnectID: a6554c4c-79a6-4db7-b7a4-201c0cf746ba 
  dataPlaneClientAuth:
    certificateSecret:
      provisioning: Manual
      secretRef:
        name: konnect-client-tls
  konnect:
    authRef:
      name: konnect-api-auth-dev-1
```